tigerpath/majors_and_certificates/majors/AAS.json

```diff
@@ -79,7 +66,7 @@
               "min_needed": 4,
               "explanation": null,
               "course_list": [
-                "AAS 201",
+                "AAS 201/PHI 291",

```

Comment:
It looks like the AAS department website went through an update after I updated this file, so the link to the page with the approved courses for each subfield is broken.

Reply:
You're right, the links are broken. Unfortunately, I think that means we'll have to wait until they fix it.

Reply:
The course list with their tracks is now listed on here: https://aas.princeton.edu/academics/all-courses/fall-2021
I've updated the course lists again according to this new page.